packages/rn-tester/js/examples/ScrollView/ScrollViewExample.js

```diff
@@ -504,6 +504,14 @@ if (Platform.OS === 'macos') {
         return <ScrollIndicatorOverlayExample />;
       },
     },
+    {
+      title: '<ScrollView> (centerContent = true)\n',
+      description:
+        'ScrollView puts its content in the center if the content is smaller than scroll view',
+      render: function (): React.Node {
+        return <CenterContentList />;
+      },
+    },
```

Comment:
Was this already there on iOS?

Reply:
Yes, this was originally here: 
https://github.com/facebook/react-native/blob/462c648b58a536cc9437d49b1f644e34d95230a1/packages/rn-tester/js/examples/ScrollView/ScrollViewExample.js#L417-L425

Reply:
Ah... One day I'll figure out how to refactor that to be more out of tree friendly.